Python/DataAugmentationUtilsPackage/DataAugmentationUtils/Embedder.py

```diff
@@ -39,9 +39,11 @@ def run_PCA(self, num_PCA):
 		eigen_vectors = np.flip(eigen_vectors, 1)
 		# get num PCA components
 		cumDst = np.cumsum(eigen_values) / np.sum(eigen_values)
+
 		if num_PCA == 0:
+			percent_variability = input("\nEnter the proportion of variability to preserve. \nFor example, to preserve at least 95% of variability enter: 0.95\n")
```

Comment:
I would add percent_variability as an input argument with a default 0.95 and use it if num_pca_modes = 0 (default)